src/node/genesis_gen.h

```diff
@@ -88,8 +88,7 @@ namespace ccf
 
     auto add_member(
       const std::vector<uint8_t>& member_cert_pem,
-      const std::vector<uint8_t>& member_keyshare_pub,
-      MemberStatus member_status = MemberStatus::ACTIVE)
+      const std::vector<uint8_t>& member_keyshare_pub)
```

Comment:
IIRC, this was written before the `ccf.service` table existed. The initial status of the member can be directly deduced from the state of the service. 


This is why many unit tests now require the `g.create_service({})` call.